[BUG] dataframe.ewm failed when has single chunk

<!--
Thank you for your contribution!

Please review https://github.com/mars-project/mars/blob/master/CONTRIBUTING.rst before opening an issue.
-->

### Describe the bug
When chunk_shape[axis] == 1 and passed a string type func, dataframe.ewm would fail
```
df = md.DataFrame({"B": [0, 1, 2, 3, 4], "A": [2, 3, 4, 5, 6]})
print(df.ewm(alpha=0.5).agg("mean").execute())
```
Got:
```
>       print(df.ewm(alpha=0.5).agg("mean").execute())

test_collect.py:75: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
../../../../core/entity/tileables.py:404: in execute
    result = self.data.execute(session=session, **kw)
../../../../core/entity/executable.py:148: in execute
    return execute(self, session=session, **kw)
../../../../deploy/oscar/session.py:1875: in execute
    return session.execute(
../../../../deploy/oscar/session.py:1669: in execute
    execution_info: ExecutionInfo = fut.result(
/Users/huanghaojie/miniconda3/envs/plotly/lib/python3.10/concurrent/futures/_base.py:458: in result
    return self.__get_result()
/Users/huanghaojie/miniconda3/envs/plotly/lib/python3.10/concurrent/futures/_base.py:403: in __get_result
    raise self._exception
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

session = <mars.deploy.oscar.tests.session.CheckedSession object at 0x147f9a260>
wait = True, show_progress = False, progress_update_interval = 1
cancelled = <asyncio.locks.Event object at 0x150b2ed40 [set]>
tileables = (DataFrame <op=DataFrameEwmAgg, key=dfc92fb1fe018893b0574bf2110c530c_0>,)
kwargs = {'warn_duplicated_execution': False}
_attach_session = <function _execute.<locals>._attach_session at 0x150b35360>
progress_bar = <mars.deploy.oscar.session.ProgressBar object at 0x150b2ecb0>
exec_task = <Task finished name='Task-260' coro=<_wrap_awaitable() done, defined at /Users/huanghaojie/miniconda3/envs/plotly/lib/python3.10/asyncio/tasks.py:643> exception=AssertionError()>
cancel_task = <Task finished name='Task-261' coro=<Event.wait() done, defined at /Users/huanghaojie/miniconda3/envs/plotly/lib/python3.10/asyncio/locks.py:201> result=True>

    async def _execute(
        *tileables: Tuple[TileableType],
        session: _IsolatedSession = None,
        wait: bool = True,
        show_progress: Union[bool, str] = "auto",
        progress_update_interval: Union[int, float] = 1,
        cancelled: asyncio.Event = None,
        **kwargs,
    ):
        execution_info = await session.execute(*tileables, **kwargs)
    
        def _attach_session(future: asyncio.Future):
            if future.exception() is None:
                for t in execution_info.to_execute_tileables:
                    t._attach_session(session)
    
        execution_info.add_done_callback(_attach_session)
        cancelled = cancelled or asyncio.Event()
    
        if wait:
            progress_bar = ProgressBar(show_progress)
            if progress_bar.show_progress:
                await _execute_with_progress(
                    execution_info, progress_bar, progress_update_interval, cancelled
                )
            else:
                exec_task = asyncio.ensure_future(execution_info)
                cancel_task = asyncio.ensure_future(cancelled.wait())
                await asyncio.wait(
                    [exec_task, cancel_task], return_when=asyncio.FIRST_COMPLETED
                )
            if cancelled.is_set():
                execution_info.remove_done_callback(_attach_session)
                execution_info.cancel()
            else:
                # set cancelled to avoid wait task leak
                cancelled.set()
>           await execution_info
E           AssertionError: [address=127.0.0.1:52298, pid=7402] Index are different
E           
E           Index classes are different
E           [left]:  Index(['B', 'A'], dtype='object')
E           [right]: MultiIndex([('B', 'mean'),
E                       ('A', 'mean')],
E                      )

../../../../deploy/oscar/session.py:1855: AssertionError
```

### To Reproduce
To help us reproducing this bug, please provide information below:

1. Your Python version
2. The version of Mars you use
3. Versions of crucial packages, such as numpy, scipy and pandas
4. Full stack of the error.
5. Minimized code to reproduce the error.

### Expected behavior
A clear and concise description of what you expected to happen.

### Additional context
Add any other context about the problem here.

--- a/mars/dataframe/window/aggregation.py
+++ b/mars/dataframe/window/aggregation.py
@@ -221,8 +221,6 @@ class BaseDataFrameExpandingAgg(DataFrameOperand, DataFrameOperandMixin):
             self._func = new_func
         elif isinstance(self._func, Iterable) and not isinstance(self._func, str):
             self._func = list(self._func)
-        else:
-            self._func = [self._func]
 
     @staticmethod
     def _safe_append(d, key, val):
@@ -274,6 +272,8 @@ class BaseDataFrameExpandingAgg(DataFrameOperand, DataFrameOperandMixin):
         chunk_cols = set(chunk_cols) if chunk_cols is not None else None
         if isinstance(op.func, list):
             op_func = {None: op.func}
+        elif isinstance(op.func, str):
+            op_func = {None: [op.func]}
         else:
             op_func = op.func
 
@@ -628,10 +628,5 @@ class BaseDataFrameExpandingAgg(DataFrameOperand, DataFrameOperandMixin):
             cls._execute_combine(ctx, op)
         else:
             in_data = ctx[op.inputs[0].key]
-            out_df = op.outputs[0]
-
             r = cls._execute_raw_function(op, in_data)
-            if out_df.ndim == 1:
-                r = r.iloc[:, 0]
-                r.name = out_df.name
             ctx[op.outputs[0].key] = r

--- a/mars/dataframe/window/ewm/tests/test_ewm.py
+++ b/mars/dataframe/window/ewm/tests/test_ewm.py
@@ -81,4 +81,4 @@ def test_ewm_agg():
     aggs = ["mean", "var", "std"]
     for a in aggs:
         r = getattr(df2.ewm(span=3), a)()
-        assert r.op.func == [a]
+        assert r.op.func == a

--- a/mars/dataframe/window/ewm/tests/test_ewm_execution.py
+++ b/mars/dataframe/window/ewm/tests/test_ewm_execution.py
@@ -37,6 +37,9 @@ def test_dataframe_ewm_agg(setup):
 
     df = md.DataFrame(raw, chunk_size=(10, 3))
 
+    r = df.ewm(alpha=0.5).agg("mean")
+    pd.testing.assert_frame_equal(r.execute().fetch(), raw.ewm(alpha=0.5).agg("mean"))
+
     r = df.ewm(alpha=0.5).agg(["mean"])
     pd.testing.assert_frame_equal(r.execute().fetch(), raw.ewm(alpha=0.5).agg(["mean"]))
 
@@ -54,6 +57,9 @@ def test_dataframe_ewm_agg(setup):
         pd.testing.assert_frame_equal(
             r.execute().fetch(), raw.ewm(alpha=0.3, ignore_na=True).agg(fun_name)
         )
+
+    r = df.ewm(alpha=0.3).agg("mean")
+    pd.testing.assert_frame_equal(r.execute().fetch(), raw.ewm(alpha=0.3).agg("mean"))
 
     r = df.ewm(alpha=0.3).agg(["mean"])
     pd.testing.assert_frame_equal(r.execute().fetch(), raw.ewm(alpha=0.3).agg(["mean"]))

--- a/mars/dataframe/window/expanding/tests/test_expanding.py
+++ b/mars/dataframe/window/expanding/tests/test_expanding.py
@@ -71,4 +71,4 @@ def test_expanding_agg():
     aggs = ["sum", "count", "min", "max", "mean", "var", "std"]
     for a in aggs:
         r = getattr(df2.expanding(3), a)()
-        assert r.op.func == [a]
+        assert r.op.func == a

